CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## [0.10.1] - 2020-06-19
+- Added Upstream Authority `vault` Plugin (#1611, #1632)
+- Improved cgroup path parsing in the `docker` workload attestor plugin (#1605)
+- Improved container ID parsing in the `k8s` workload attestor plugin (#1649)
```

Comment:
These should be worded similarly since they both involve parsing the container ID from the cgroup path

Reply:
(I recognize the PRs themselves didn't have consistent wording :) )